setup.py

```diff
@@ -1,13 +1,24 @@
+import codecs
 import os
-import sys
+import re
 
 from setuptools import setup
 
-# Add the package source
-sys.path.append(os.path.join(os.path.dirname(__file__), 'src'))
+here = os.path.abspath(os.path.dirname(__file__))
 
-# Import our package
-import iceqube  # noqa
```

Comment:
So the reason why we suddenly cannot load `iceqube` is because its `__init__` was changed. Other modules are now being imported in `iceqube.__init__`, this is "considered harmful".

https://github.com/learningequality/iceqube/commit/bfbb929bb6e0f06585baa37a7d60a08d7cf06516

Ind hindsight, if you were looking for a quick fix, I would have suggested to maintain the version string in `setup.py` instead of `__init__.py`.

Regarding imports in `__init__.py`, we trigger a "mandatory" load of sub modules, which can be really nasty in situations where they're not supposed to be loaded... for instance in this case, where we just wanted the version string and instead go loads of unavailable libraries. It also becomes impossible to get around it when importing other modules from `iceqube.*`, because `iceqube.__init__` is always called.

I would suggest for 0.4 to move these imports out of `iceqube.__init__` so it can load normally and avoid a `setup.py` that traverses folders and pattern matches files.